packages/atoms/src/ListItem/ListItemText.tsx

```diff
@@ -0,0 +1,30 @@
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Text } from '../Text';
+import { View } from 'react-native';
+import React from 'react';
+import { BaseListItem } from './listItemTypes';
+
+type ListItemTextProps = Pick<BaseListItem, 'title' | 'subtitle' | 'wrap'>;
```

Comment:
What does the wrap mean pls?

Reply:
**wrap** is an analogy to _flex-wrap: wrap_ to wrap long text onto multiple lines in this case. Maybe it's unnecessarily parameterized and wrap = false (text is forced onto one line and the rest which is not visible is filled with dots) is not necessary at all for our use cases.

The image below shows the current state.

![Snímek obrazovky 2022-05-26 v 15 29 06](https://user-images.githubusercontent.com/2227592/170497480-2d2f3725-f924-432a-9f80-c80de527d51f.png)

But wrap = true is the default case as I can see on Figma screens so I will probably set it wrap = true by default in component props:

![Snímek obrazovky 2022-05-26 v 15 32 44](https://user-images.githubusercontent.com/2227592/170497966-e57018e0-ca88-4722-9c3f-8342814034af.png)

Reply:
Maybe I should rename it to **wrapText** to be more clear.